src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -261,14 +261,14 @@ extern "C" HRESULT CallApplicationMain(PCALL_APPLICATION_MAIN_DATA data)
 
     hr = executeAssembly(pathToBootstrapper.c_str(),
                          coreClrDllPath.c_str(),
-                         "dnx.coreclr.managed",
+                         "Microsoft.Dnx.Host.CoreClr",
                          sizeof(property_keys) / sizeof(property_keys[0]),
                          property_keys,
                          property_values,
                          data->argc,
                          (const char**)data->argv,
                          nullptr,
-                         "dnx.coreclr.managed, Version=0.1.0.0",
+                         "Microsoft.Dnx.Host.CoreClr, Version=0.1.0.0",
```

Comment:
nit: I changed the version to `0.0.0.0` in the windows bootstrapper. The version seems to be required even though it appears it is not being used.